### PR TITLE
Closes #27 - Closes #15 - Closes #4

### LINKED ISSUE #4: Revision of the button for the view of metrics.
The button that can be used to navigate to the metrics view is to be revised. User tests show that the metrics icon is not recognizable as a button. Changes should be made in this regard.

### LINKED ISSUE #15: Fix the back button of the minikube metrics
when clicking on the minikube component (overview -> node -> minikube) , a metrics window will be opened. but when reaching this point, the back button doesn't work.

### LINKED ISSUE #27: Update UI components to use Grafana components and styles
A lot of the components used right now can be replaced by Grafana components (grafana/ui).
The colors and schemes could be used directly from Grafana as well. Thatfore an upgrade from CSS to SASS is probably the best option, because it allows us to import SASS variables from Grafana.

### PR DESCRIPTION
**Pull request by leoschur**
Closes #27, #15 and #4

- refactured drilldown,
- exchanged components to grafana ui components,
- migrated css to emotion

### CODE REVIEW

File: src/SimplePanel.tsx [hunk 1]
```diff
@@ -122,31 +125,19 @@ export const SimplePanel: React.FC<Props> = ({ options, data, width, height, tim
...
-
-  };
-
-  /**
-   * Is called to close the drilldown menu.
-   */
-  const closeDrilldown = () => {
-    console.log('hey');
-    setShowDrilldown(false);
   };
```
Comment (salim-1997) [inline src/SimplePanel.tsx:133]: is this function replaced with something else?

File: src/NodeMetric.tsx [hunk 2]
```diff
@@ -57,7 +58,13 @@ export const NodeMetric = ({ width, height, data, timeRange, setShowGraph, focus
...
           </div>
         </div>
-        <div className="NodeGraph">
+        <div
+          className={css`
+            width: 100%;
+            display: flex;
+            justify-content: center;
+          `}
+        >
```
Comment (salim-1997) [inline src/NodeMetric.tsx:67]: couldn't this one be also made with the styles class like the other (zb. styles.nodeGraph)? or do you think it's better with emotion?

File: src/NodeMetric.tsx [hunk 3]
```diff
@@ -38,16 +41,14 @@ export const NodeMetric = ({ width, height, data, timeRange, setShowGraph, focus
...
   return (
     <div>
-      <img
-        src="https://raw.githubusercontent.com/fylip97/Thesis/main/src/img/back2.png"
-        onClick={() => setShowGraph(false)}
-        className="graphBack"
-      />
+      <Button variant="secondary" size="sm" icon="arrow-left" onClick={() => setShowGraph(false)}>
+        Back
+      </Button>
```
[the remark this answers is not in the document]
Comment (leoschur) [reply src/NodeMetric.tsx:46]: image
it´s not overwritten but added in front or do you get a different result?

[the remark this answers is not in the document]
Comment (salim-1997) [reply src/NodeMetric.tsx:47]: it's actually good like that. i was only considering the fact that if the screen was too small ,the icon and the string will over lapse on each other but i think that can't happen, we can keep like that.

**Pull request merged**